src/bag3_digital/layout/stdcells/inv_diff.py

```diff
@@ -51,8 +52,9 @@ def get_default_param_values(cls) -> Mapping[str, Any]:
             w_n=0,
             ridx_p=-1,
             ridx_n=0,
+            chain=False,
             dummy_dev=False,
-            sig_locs=None,
+            sig_locs=dict(),
```

Comment:
Default value of sig_locs has to be None because empty dictionaries are not hashable. In the code, replace sig_locs with empty dict if it's None.